azkaban/__main__.py

```diff
@@ -10,7 +10,7 @@
   azkaban run [-jkp PROJECT] [-a ALIAS | -u URL] [-b | -m MODE] [-e EMAIL ...]
               [-o OPTION ...] FLOW [JOB ...]
   azkaban schedule [-jkp PROJECT] [-a ALIAS | -u URL] [-b | -m MODE]
-                   [-e EMAIL ...] [-o OPTION ...] [-s SPAN] (-d DATE) (-t TIME)
+                   [-e EMAIL ...] [-o OPTION ...] [-s SPAN] [-n] (-d DATE) (-t TIME)
```

Comment:
I'd be happy to, but wouldn't it read better if it was [-jkn] [-p PROJECT]? I was a bit confused by that bit when I was starting this task.

Reply:
Thanks for consolidating. Agreed that it might be more intuitive to separate the nullary from unary flags but let's not expand the scope of this change. Updating all the sites could be a nice separate commit.